src/config/settings.js

```diff
@@ -38,17 +38,34 @@ function load() {
   let settings;
   try {
     settings = JSON.parse(rawdata);
-  } catch(_) {
-    settings = {}
+  } catch (_) {
+    save(defaultSettings);
+    return defaultSettings;
   }
-  // clone settings because we check to see if the returned value is different
   let cfg = updateSettings(Object.assign({}, settings));
   if (!isDeepEqual(cfg, settings)) {
     save(cfg);
   }
   return cfg;
 }
 
+/**
+ * Verify that the loaded version of kubernetes, if specified, is in the current list of supported versions.  Throw an exception if not.
+ * @param{Object} settings
```

Comment:
```suggestion
 * @param {Settings} settings
```
Just because we can. (I'm happy to merge without.)

Reply:
Could these go in a separate doc for post-merge changes?